webpack.config.js

```diff
@@ -123,28 +123,90 @@ module.exports = {
                 test: /\.(jsx|js)$/
             },
             {
-                test: /\.(scss|css)$/,
+                test: /\.less$/,
```

Comment:
Are these just for our remaining three lib/*.less files (imported from cockpit)? (listing, variables, form-layout), or do we also need less for some node_modules/ imports? IOW, can we drop this once we convert these three lib files to scss in cockpit and sync them over?

Reply:
We can drop this once we sync the following:
```
[kkoukiou@sequioa cockpit-podman]$ find lib/ -name *.less
lib/variables.less
lib/form-layout.less
lib/listing.less
[kkoukiou@sequioa cockpit-podman]$ find src/ -name *.less
src/css-overrides/toast.less

```
I am happy to do that in seperate PR.